components/Comment/Comment.tsx

```diff
@@ -240,13 +255,13 @@ const Comment = ({ comment, document, ignoreChildren }: CommentArgs) => {
                     setIsEditMode(false);
                   }
                 }}
+                handleCancel={() => _handleCloseEdit()}
```

Comment:
just for code style, if we're not passing anything into the function, should we instead do 

`handleCancel={_handleCloseEdit}`

Reply:
@lightninglu10 you are right. Also, I've discovered that in some circumstances, inline anonymous functions will cause another render which is not always trivial! For example, in the case of react-pdf, passing inline functions like this causes another re-render of the page